### ISSUE TITLE
Terminology: to be discussed with political scientist

### ISSUE THREAD

**Issue by matyaskopp** [clarin-eric/ParlaMint#220]
This issue is to collect a list of terms and situations where we are unsure what the proper terminology is.

### members of parliament, chambers, houses...

Currently, we are using MP for parliament members, but this is a UK-based naming. It is correct for the house of commons in the UK parliament, but not proper for the house of lords and probably not for other countries for its UK bias (discussed here: #219 )

Suggested values: "member", "legislator", ...

**answer:** ???

**Comment by TomazErjavec** [comment]
We have now arranged a meeting with some political scientists (or at least historians) on Tuesday May 10th at 12:00. I will ask them about our open problems there, but we will also have a Zoom connection, @matyaskopp and @coltekin, you are very welcome to attend as well!
In any case, it would be good if we made some slides about the open problems (with links to open issues), but for discussing this email will be better.
